changelog/1726.trivial.rst

```diff
@@ -0,0 +1,2 @@
+Changed `~plasmapy.formulary.quantum.chemical_potential` to use the
+Broyden-Fletcher-Goldfarb-Shanno algorithm to implicitly solve for beta.
```

Comment:
```suggestion
Changed `~plasmapy.formulary.quantum.chemical_potential` to use the
Broyden-Fletcher-Goldfarb-Shanno algorithm to implicitly solve for 
the β(\ :math:`= 1/(k_B T)`\ ) parameter.
```

Only so beta is not mistaken for the plasma beta, which would likely be common in this community.

Reply:
Whoops, that shouldn't say beta anyways. It should be the dimensionless chemical potential